src/main/java/io/reactivex/Observable.java

```diff
@@ -1471,6 +1480,7 @@ public static int bufferSize() {
      * @param <T> the element type
      * @param source the emitter that is called when an Observer subscribes to the returned {@code Observable}
      * @return the new Observable instance
+     * @throws NullPointerException if source is null
```

Comment:
I'm not sure if these need to be spelled out, but I guess other places have them already...